src/index.js

```diff
@@ -1,166 +1,143 @@
+        
         <i class="fa-solid fa-trash-can" id"${index}"></i>
+        <div class="button-details">
         <button class="edit" id="${index}">edit</button>
         <button type="button" id="${index}" class="delete">del</button>
+        </div>
+        </div>
               `;
-    
-      return divElement;
-      
-      
-    
-    };
-  
-    inputSubmit.addEventListener('click', (e) => {
-      e.preventDefault();
-        if(inputDiv.value.trim()!== ''){
-          if(isEdit === false) {
-                      const list = {
-              description: inputDiv.value,
-              completed: false,
-              index: todoList.length +1, 
-          };
-
-          todoList.push(list);
-          setLocalStorage(todoList);
-          todoList = getLocalStorage();
-          divContainer.innerHTML='';setLocalStorage(todoList);
-          todoList.forEach((item)=>{
-            divContainer.append(displayTodo(item));
-          })
-          } else {
-            todoList[editId].description = inputDiv.value
-            setLocalStorage(todoList);
-            todoList = getLocalStorage();
-            divContainer.innerHTML='';
-            todoList.forEach((item)=>{
-              divContainer.append(displayTodo(item));
-            })
-            isEdit = false;
-            
-          }
-          inputDiv.value = ``;
-          removeEvents();
-         editEvents();
-         
-          }; 
-    
-    });
-  
-    
-    todoList = getLocalStorage();
-    divContainer.innerHTML='';
-    todoList.forEach((item)=>{
-      divContainer.append(displayTodo(item));
-
-    
-    })
-    const togle = document.querySelectorAll('.complete');
-    togle.forEach((toggleItem, index)=> {
-      toggleItem.addEventListener('change', ()=>{
-        let todoList = getLocalStorage();
-        todoList[index].completed= todoList[index].completed ? false : true;
-        setLocalStorage(todoList);
-      })
-    })
-    removeEvents()
-    editEvents() 
-  
-    
-    
-
-    
\ No newline at end of file
+
+  return divElement;
+};
+const editEvents = () => {
+  document.querySelectorAll('.edit').forEach((button) => button.addEventListener('click', (e) => {
+    e.preventDefault();
+    let id;
+    if (button.id > 0) {
+      id = button.id - 1;
+    } else {
+      id = 0;
+    }
+
+    isEdit = true;
+    editId = id;
+
+    inputDiv.value = todoList[id].description;
+    inputDiv.focus();
+  }));
+};
+
+const reassignIndex = (todoList) => {
+  todoList.forEach((item, i) => {
+    item.index = i + 1;
+  });
+};
+
+const removeEvents = () => {
+  document.querySelectorAll('.delete').forEach((button) => button.addEventListener('click', (e) => {
+    e.preventDefault();
+    let id;
+    if (button.id > 0) {
+      id = button.id - 1;
+    } else {
+      id = 0;
+    }
+
+    deleteTask(id); // eslint-disable-line
```

Comment:
- [ ] Kindly remove the disable linters line as linters help us to write clean code.